integration/apps/rack/Gemfile

```diff
@@ -23,10 +23,10 @@ gem 'resque'
 gem 'rake'
 
 gem 'dogstatsd-ruby'
-# Choose correct specs for 'ddtrace' demo environment
-gem 'ddtrace', *Datadog::DemoEnv.gem_spec('ddtrace')
+# Choose correct specs for 'datadog' demo environment
+gem 'datadog', *Datadog::DemoEnv.gem_spec('datadog')
 
-# Needed for ddtrace profiling
+# Needed for datadog profiling
 google_protobuf_versions = [
```

Comment:
Minor: Actually no longer needed ;)

Reply:
As in delete these arrays? Can you make a code suggestion? I can apply it once I see the pattern of it.

Reply:
We can clean it all, the dependency is no longer needed:

```diff
-# Needed for ddtrace profiling
-google_protobuf_versions = [
-  '~> 3.0',
-  '!= 3.7.0.rc.2',
-  '!= 3.7.0.rc.3',
-  '!= 3.7.0',
-  '!= 3.7.1',
-  '!= 3.8.0.rc.1',
-  '!= 3.24.0.rc.1',
-]
-if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.4')
-  gem 'google-protobuf', *google_protobuf_versions
-else
-  # Bundler resolves incorrect version (too new, incompatible with Ruby <= 2.3)
-  gem 'google-protobuf', *google_protobuf_versions, '< 3.19.2'
-end
-
```